pysatMagVect/tests/test_core.py

```diff
@@ -556,7 +556,7 @@ def test_unit_vector_plots(self):
                 self.inst[:,'sc_xhat_x'], self.inst[:,'sc_xhat_y'], self.inst[:,'sc_xhat_z'] = 1., 0., 0.
                 self.inst[:,'sc_yhat_x'], self.inst[:,'sc_yhat_y'], self.inst[:,'sc_yhat_z'] = 0., 1., 0.
                 self.inst[:,'sc_zhat_x'], self.inst[:,'sc_zhat_y'], self.inst[:,'sc_zhat_z'] = 0., 0., 1.
-                self.inst.data.index = pysat.utils.season_date_range(pysat.datetime(2000,1,1),
+                self.inst.data.index = pysat.utils.time.create_date_range(pysat.datetime(2000,1,1),
```

Comment:
I think we need to be more careful changing method names in pysat in the future. The public version of pysat doesn't have this call. Perhaps we should have a method with the old name, for at least a version, that raises a Deprecation Warning and transparently calls the new routine. Technically, we can't release this code for use until the next version of pysat.

Reply:
Good catch.